app/controllers/miq_ae_tools_controller.rb

```diff
@@ -198,6 +198,7 @@ def review_import
   def retrieve_git_datastore
     redirect_options = {:action => :review_git_import}
     git_url = params[:git_url]
+    verify_ssl = params[:git_verify_ssl]  == "true" ? OpenSSL::SSL::VERIFY_PEER : OpenSSL::SSL::VERIFY_NONE 
```

Comment:
There's an extra space in the `params[:git_verify_ssl] == "true"` part, looks kinda off.